app/src/components/UserProfile.js

```diff
@@ -15,14 +19,51 @@ export default function UserProfile() {
     }));
   };
 
-  const handleSubmit = (event) => {
+  const updateUser = async (event) => {
     event.preventDefault();
-    console.log(state);
+    const token = await getAccessTokenSilently();
```

Comment:
I didn't think to make this variable outside of this function definition, i'll pull it out and see if i can refactor access to the token as needed